### PR TITLE
Account change information cp updates

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-24eefad556/how-to/). :bow:

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-24eefad556/how-to/).

**Comment by cyrichardson** [comment]
Suggestions here: 
[change-account-information_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1759950/change-account-information_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-4c068881ee/how-to/). :bow:

File: content/general/change-account-information.md [hunk 1]
```diff
@@ -22,19 +22,25 @@ both the assignor (original account owner) and the assignee (new account owner).
 You can download the **Service Transfer** form directly from
 the [Cloud Control Panel](https://mycloud.rackspace.com/) by using the following steps.
 
-**Note:** If you are a Dedicated Hosting customer, you can request this form
-directly from your Account Manager.
+**Note:** If you are a Dedicated Hosting customer, you can request all of the forms described in this article from your account manager.
```
Comment (stephamon) [inline content/general/change-account-information.md]: You need a blank line after the note so the following numbered list doesn't get pulled into it as a paragraph.

**Comment by stephamon** [comment]
👍

**Pull request merged**